pcsx2/gui/MainMenuClicks.cpp

```diff
@@ -806,6 +806,21 @@ void MainEmuFrame::Menu_Capture_Screenshot_Screenshot_Click(wxCommandEvent & eve
 	GSmakeSnapshot(g_Conf->Folders.Snapshots.ToAscii());
 }
 
+void MainEmuFrame::Menu_Capture_Screenshot_Screenshot_As_Click(wxCommandEvent &event)
+{
+	if (!CoreThread.IsOpen())
+	{
+		return;
+	}
```

Comment:
Personally I wouldn't add brackets for 1 line code, 2 less lines. There are others similar through the pr.